pyproject.toml

```diff
@@ -1,21 +1,61 @@
+[tool.black]
```

Comment:
https://github.com/RedHatQE/rosa-python-client/pull/71#pullrequestreview-1558368463 is not addressed